deploy/kustomization.yaml

```diff
@@ -5,7 +5,7 @@ namespace: fleet-system
 resources:
 - ./component_namespace.yaml
 - ./service_account.yaml
-- ./clusterrole_binding.yaml
+- ./admin_clusterrole_binding.yaml
```

Comment:
nit: this is not really admin though.

Reply:
We have two cluster roles: admin & work; the former being a out-of-box ClusterRole, the latter being our custom ClusterRole. I agree this file is overloaded and is doing more than binding the admin role to the work-controller-sa.

This should be split into separate files, but we were the make deploy kustomize throws an error when I split them into separate files. I will have that figured out push an update.